src/event-decoder/metabonding/metabonding.event.ts

```diff
@@ -16,13 +15,22 @@ export class MetabondingEvent extends RawEvent {
         super(init);
         Object.assign(this, init);
         this.decodedTopics = new MetabondingEventTopics(this.topics);
-        const decodedEvent = this.decodeEvent();
-        this.userEntry = new UserEntry({
-            tokenNonce: decodedEvent.tokenNonce.toNumber(),
-            stakedAmount: decodedEvent.stakeAmount.toFixed(),
-            unstakedAmount: decodedEvent.unstakeAmount.toFixed(),
-            unbondEpoch: decodedEvent.unbondEpoch.toNumber(),
-        });
+
+        if (this.data != '') {
```

Comment:
How about `!data`? (in case the partial init comes with `data: null`).

Reply:
done